torchao/quantization/prototype/mixed_precision/scripts/quant_model_size.py

```diff
@@ -0,0 +1,35 @@
+def quantized_model_size_in_bytes(num_elements, group_size, bit_zeropoint, bit_scale, x, y, A, B):
+    # Size for A-bit quantization layers
+    size_A_bit = x * (num_elements * A + num_elements // group_size * (bit_zeropoint + bit_scale))
+    
+    # Size for B-bit quantization layers
+    size_B_bit = y * (num_elements * B + num_elements // group_size * (bit_zeropoint + bit_scale))
+    
+    # Total quantized model size in bits
+    total_size_bits = size_A_bit + size_B_bit
+    
+    # Convert to bytes
+    total_size_bytes = total_size_bits / 8
+    
+    # Convert to gigabytes
+    total_size_gb = total_size_bytes / (1024 ** 3)
+    
+    return total_size_gb
+
+# Example usage
+num_elements = 250945664 #number of elements per Llama3 linear layer
```

Comment:
can this be calculated from the model instead of hardcoded? also I feel a better integration is just to fix and extend https://github.com/pytorch/ao/blob/5787e9e73f6493765052e3638bd0ae9103496bae/torchao/utils.py#L188

Reply:
Yes this is a temporary solution for Llama3. Thanks for the suggestion! I will try to generalize it by extend the get_model_size_in_bytes.